app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -268,6 +257,27 @@ public void onChanged(Integer progress) {
         return view;
     }
 
+    /**
+     * Gets the current state of the application and updates the cursor state accordingly.
+     *
+     * Note that this pattern could use some improvements:
+     * - It's a little weird to get the current state from globals, rather than get passed in relevant values.
+     * - BrowserFragment.setCursorEnabled should be called from this code path, but that's unclear
```

Comment:
mmm that's a good point - I think that's one of the things we need to get right in our refactor.